.editorconfig

```diff
@@ -9,6 +9,10 @@ charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
 
+[*.js]
```

Comment:
Why this, it has nothing to do with the rest of the PR.

Also should be discussed if we want to keep tabs here, makes switching contexts between PHP and JS harder IMHO.